psl-clj/README.md

```diff
@@ -0,0 +1,106 @@
+# Example
+
+Install [Leiningen](https://leiningen.org).  Then, create a new example application called `myproj` from the `template` directory.
```

Comment:
I know pretty nothing about this dependency except this sentence and the first sentence on their homepage, but it there some maven module that we could use instead of having them install it externally?
This one came up: https://mvnrepository.com/artifact/leiningen/leiningen
Is it at all related?

Reply:
In psl-clojure/pom.xml, we use a maven plugin to compile psl-clojure itself, so Leiningen is not needed to compile psl-experimental.  Users of psl-clojure use Leiningen to generate a new project from the template (like an archetype).  They also use Leiningen to develop/interact with their project via its REPL, to package it into a jar, etc.